src/py/arc_allocator.rs

```diff
@@ -38,10 +42,12 @@ impl Clone for ArcSExp {
 
 impl ArcAllocator {
     pub fn new() -> Self {
-        ArcAllocator {}
+        ArcAllocator {
+            cumulative_allocations: 0,
```

Comment:
This won't work because `ArcAllocator` objects interoperate.

Reply:
there's more than one allocator?